nav2_msgs/action/Wait.action

```diff
@@ -3,6 +3,7 @@ builtin_interfaces/Duration time
 ---
 #result definition
 builtin_interfaces/Duration total_elapsed_time
+uint16 error_code
```

Comment:
This feels unnecessary if there's no error codes

Reply:
Yeah, but anything using the timed behavior needs a error code.